rkt/api_service.go

```diff
@@ -82,6 +82,14 @@ func (s *v1AlphaAPIServer) GetInfo(context.Context, *v1alpha.GetInfoRequest) (*v
 			RktVersion:  version.Version,
 			AppcVersion: schema.AppContainerVersion.String(),
 			ApiVersion:  supportedAPIVersion,
+			GlobalFlags: &v1alpha.GlobalFlags{
+				Dir:                globalFlags.Dir,
```

Comment:
If I understand this correctly, the intention of this PR is expose the options used to call the rkt API service through the API service itself. Then, something (the kubelet) can query the API service and use those options when spawning new pods.

Do we want the options passed to the API service (`globalFlags.Dir`) or the real data directory `getDataDir()` (since it can return a directory that depends on the rkt config dirs if we don't pass the `--dir` flag).

I guess this doesn't really matter because you're not gonna have different rkt configurations in different nodes?

Am I missing something?